retrodeck.sh

```diff
@@ -56,10 +56,10 @@ https://retrodeck.net
       ;;
     --reset-component*)
       echo "You are about to reset one or more RetroDECK components or emulators."
-      echo "Available options are: es-de, retroarch, cemu, dolphin, duckstation, gzdoom, melonds, pcsx3, pico8, ppsspp, primehack, rpcs3, ryujinx, xemu, vita3k, mame, all"
+      echo "Available options are: es-de, retroarch, cemu, dolphin, duckstation, gzdoom, melonds, pcsx3, pico8, ppsspp, primehack, rpcs3, ryujinx, xemu, vita3k, ruffle, mame, boilr, all"
```

Comment:
Boilr can be removed

Reply:
I am not sure now that got there!

Reply:
The Steam Rom manager branch was megerd but we realized ti was not read, so I removed all the references, but this was not yet removed while you clone probably.